cluster/libvirt-coreos/util.sh

```diff
@@ -51,6 +51,16 @@ function detect-minions {
   KUBE_MINION_IP_ADDRESSES=("${MINION_IPS[@]}")
 }
 
+# Creates apiserver parameter for apis extension support.
+# If RUNTIME_CONFIG contains extensions/v1beta1 modify parameter for api-server
+RUNTIME_CONFIG=${RUNTIME_CONFIG:-""}
+function detect-api-extension {
+    API_VERSION="api/v1"
+    if [[ $RUNTIME_CONFIG = *extensions/v1beta1* ]]; then
```

Comment:
I am sorry I dont understand, who is setting RUNTIME_CONFIG env var?
Can we directly use --runtime-config=${RUNTIME_CONFIG}?
Note that runtime-config specifies the overrides, so setting runtime-config="" will not disable v1.
You need to pass runtime-config="api/v1=false" to turn it off.